### PR TITLE
Apidocs cleanup

### PR DESCRIPTION
**Pull request by sharmanat0r**
Have tried to simplify the docs, removed references, and reworded a bunch of things that I found confusing.

# API References removed
Due to being part of the old Membership Platform and some being generally low usage that we don't want newer clients jumping onto:

## Agreements
- POST /agreements
- POST /agreements/{agreement_ref}/accept
- POST /agreements/{agreement_ref}/decline

## Contacts
- POST /contacts

## Payment Requests
- GET /payment_requests/incoming
- POST /payment_requests/{payment_request_ref}/approve
- POST /payment_requests/{payment_request_ref}/decline
- GET /payment_requests/{payment_request_ref}/history

## Payouts
- POST /payouts/{ref}/retry

## Refunds
- GET /refunds/incoming

# Removed information on:
- precheck_funds:
   - we had too many merchants using this flag without guidance and not understanding how it worked.

# Simplified areas:
- Payment batches
   -  to only have 1 payout object per batch
- Simplified Refunds
   - This area was confusing to me as it talks about refunding Payouts....But we only refund Payment Requests
- Simplified Payouts
   - What is a Payout....This area was grey to everyone I asked so I tested it, simplified the wording and marked some of it is Deprecated and advised not to use it
- Deleted all unused Schemas from the removed endpoint references
- Added missing channel attribute to Transaction schema

### CODE REVIEW

File: source/openapi3/split.yaml [hunk 1]
```diff
@@ -2,25 +2,19 @@ openapi: 3.0.0
 info:
   title: Split API
   description: >
-    Split allows you to make, get and manage payments using nothing but bank
-    accounts.
+    Split enables your business to make, collect and receive payments using nothing but bank accounts.
+    We've overlayed a simple REST-ful API on top of the legacy (and newer) Australian payment rails,
```
Comment (zubin) [inline source/openapi3/split.yaml]:
```suggestion
    We've overlaid a simple REST API on top of the legacy (and newer) Australian payment rails, 
```

File: source/openapi3/split.yaml [hunk 2]
```diff
@@ -393,14 +387,12 @@ info:
     ## Making payments
 
     In order to payout funds, you'll be looking to use the
```
Comment (zubin) [inline source/openapi3/split.yaml]: Are we still using the term _payouts_? Or can we simplify this further, eg:

```suggestion
    To make payouts, use the
```

File: source/openapi3/split.yaml [hunk 3]
```diff
@@ -898,7 +830,7 @@ info:
     If your integration includes allowing us to pre-fail transactions prior to being processed, you may want to test that your system is handling these events correctly. A transaction will pre-fail when the available balance of the customers account is less than the amount of the payment being requested. This is checked during pre-processing just before your debit is sent for processing if there is an active bank connection.
 
 
-    In the Sandbox environment, if the contact you are attempting to debit has a bank connection that was created through our Instant Account Verification feature, the available balance of their bank account will always be `$123.45`. Any payment requests above this amount will pre-fail and any amount less than or equal to this amount will succeed.
+    In the Sandbox environment, if the contact you are attempting to debit has a bank connection that was created through our Instant Account Verification feature, the available balance of any <strong>Transactional</strong> bank account will always be `$123.45`. Any payment requests above this amount will pre-fail and any amount less than or equal to this amount will succeed.
```
Comment (zubin) [inline source/openapi3/split.yaml]: Consider wrapping this long line so it's easier to read.

File: source/openapi3/split.yaml [hunk 4]
```diff
@@ -1631,13 +1565,15 @@ tags:
...
 
+      Supported payment rails:
 
-      1. General details about the Payment.
 
-      2. One or many Payouts with individual recipients, amounts and
-      descriptions.
+      * NPP: Real-time payments
+
+      * DE: Slower payments
```
Comment (zubin) [inline source/openapi3/split.yaml]: Should we indicate what the acronyms stand for?

File: source/openapi3/split.yaml [hunk 5]
```diff
@@ -1847,12 +1759,12 @@ tags:
             "parent_ref": null,
             "type": "debit",
             "category": "payout_refund",
-            "created_at": "2016-12-07T23:15:00Z",
-            "matures_at": "2016-12-10T23:15:00Z",
+            "created_at": "2021-04-07T23:15:00Z",
+            "matures_at": "2021-04-10T23:15:00Z",
```
Comment (zubin) [inline source/openapi3/split.yaml:1800]: Modernised! 😁

File: source/openapi3/split.yaml [hunk 6]
```diff
@@ -1900,7 +1815,7 @@ tags:
 
 
       <aside class="notice">
-        The <code>user_voided</code> and <code>admin_voided</code> <code>failure_reason</code>s can sometimes be accompanied by the <code>failure_details</code> key which includes user submitted comments relating to the <code>failure_reason</code>.
+        The <code>user_voided</code> and <code>admin_voided</code> <strong>failure_reasons</strong> can sometimes be accompanied by the <strong>failure_details</strong> key which includes user submitted comments relating to the <strong>failure_reason</strong>.
```
Comment (zubin) [inline source/openapi3/split.yaml]: The `<code>` tag seems appropriate. Am curious why this was changed. 🤔

File: source/openapi3/split.yaml [hunk 7]
```diff
@@ -2309,12 +2090,12 @@ paths:
 
         <aside class="notice">
           To enable this feature, please contact our support team with the following information:
-            <li>Your full legal account name</li>
+            <li>Your full legal business name</li>
             <li>A legally owned domain name: for your PayID email addresses</li>
-            <li><strong>alias_name</strong>: the short business name that will be displayed to your customers</li>
+            <li><strong>alias_name</strong>: a shortened business name that will be displayed to your customers upon PayID resolution</li>
```
Comment (zubin) [inline source/openapi3/split.yaml]: What if the business name is already short?
```suggestion
            <li><strong>alias_name</strong>: a potentially shortened business name that will be displayed to your customers upon PayID resolution</li>
```

File: source/openapi3/split.yaml [hunk 8]
```diff
@@ -2536,7 +2294,7 @@ paths:
       tags:
         - Contacts
       summary: Refresh contact bank connection
-      description: Request the bank connection for a contact to refresh available funds. This is intended for use in conjunction with the `precheck_funds` option, see [Payment Request - Precheck Funds](/#precheck-funds-lifecycle)
+      description: Refresh one of your contact's bank connections to get the latest available balance.
```
Comment (zubin) [inline source/openapi3/split.yaml]: Are we capitalising actors?
```suggestion
      description: Refresh one of your Contact's bank connections to get the latest available balance.
```

File: source/openapi3/split.yaml [hunk 9]
```diff
@@ -2731,16 +2489,21 @@ paths:
...
-        This endpoint allows you to retry the payout without having to recreate the parent request. e.g A Payment or Payment Request.
+        <aside class="notice">
+          Rather than using this endpoint, we now recommend creating a brand new transaction as this simplifies and maintains data integrity for the lifecycle of each attempt
+        </aside>
       operationId: RetryAPayout
       parameters:
         - name: ref
           in: path
-          description: Payout debit or credit reference
+          description: Payment Request debit or credit reference number e.g C.2 / D.72
```
Comment (zubin) [inline source/openapi3/split.yaml]: Shall we use bigger numbers to give a more accurate indication of what refs will look like? 🤷

File: source/openapi3/split.yaml [hunk 10]
```diff
@@ -2880,175 +2642,46 @@ paths:
...
+          <li>Must be applied against a successfully cleared Payment Request (credit)</li>
+          <li>Many refunds may be created against the original Payment Request. The total refunded amount must not exceed the original value</li>
+        </ul>
+      operationId: IssueARefund
       parameters:
-        - name: payment_request_ref
+        - name: credit_ref
           in: path
-          description: 'Single value, exact match'
+          description: 'The credit reference number e.g C.625v'
```
Comment (zubin) [inline source/openapi3/split.yaml:2700]: Is the trailing `v` in the ref intentional?

File: source/openapi3/split.yaml [hunk 11]
```diff
@@ -3865,34 +3304,77 @@ components:
...
+    PerPayout:
+      title: Per payout terms
+      required:
+        - min_amount
+        - max_amount
+      type: object
+      properties:
+        min_amount:
+          type: integer
+          description: Minimum amount in cents a PR can be in order to be auto-approved. Specify <code>null</code> for no limit.
```
Comment (zubin) [inline source/openapi3/split.yaml]: Feel free to ignore, as maybe it's just me, but may I suggest avoiding `PR` acronyms? To me (and possibly other devs) PR means "pull request" and therefore this adds cognitive overhead to the docs.

File: source/openapi3/split.yaml [hunk 12]
```diff
@@ -4631,7 +4061,7 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Payout'
-          description: One or many Payouts
+          description: One Payout object only
```
Comment (zubin) [inline source/openapi3/split.yaml:4103]: Are we moving away from the term _payout_?

**Review by zubin — APPROVED** [review]
Epic! 👏 🏅 

Lots of non-blocking suggestions.

File: source/openapi3/split.yaml (hunk 5 above)
Comment (sharmanat0r) [reply source/openapi3/split.yaml:1800]: Haha totally, there are so many of these so ill update the dates and schemas in another PR :)

File: source/openapi3/split.yaml (hunk 9 above)
Comment (sharmanat0r) [reply source/openapi3/split.yaml]: Absolutely, and will address this when we modernize the dates  :)

File: source/openapi3/split.yaml (hunk 10 above)
Comment (sharmanat0r) [reply source/openapi3/split.yaml:2700]: Yep, was trying to modernize the new descriptions but its a whole other activity and will make these more random and longer next PR

File: source/openapi3/split.yaml (hunk 11 above)
Comment (sharmanat0r) [reply source/openapi3/split.yaml]: Agree!

File: source/openapi3/split.yaml (hunk 12 above)
Comment (sharmanat0r) [reply source/openapi3/split.yaml:4103]: Yeh I'm trying to clean up Payout references as its confusing when we talk about Payments. But Payout should mean the same thing as a Payment, but sometimes Payouts have been used in the context of Payment Request which is madness

File: source/openapi3/split.yaml [hunk 13]
```diff
@@ -2,25 +2,19 @@ openapi: 3.0.0
 info:
   title: Split API
   description: >
-    Split allows you to make, get and manage payments using nothing but bank
-    accounts.
+    Split enables your business to make, collect and receive payments using nothing but bank accounts.
+    We've overlaid a simple REST API on top of the legacy (and newer) Australian payment rails, 
+    enabling all of your account to account payment flows through 1 simple integration.
```
Comment (tomdalling) [inline source/openapi3/split.yaml]: I usually suggest hyphenating phrasal adjectives.

```suggestion
    enabling all of your account-to-account payment flows through 1 simple integration. 
```

File: source/openapi3/split.yaml [hunk 14]
```diff
@@ -737,13 +715,16 @@ info:
 
     # Sandbox Testing Details
 
-    Try out your happy paths and not-so happy paths, the sandbox is a great place to get started without transferring actual funds. All transactions are simulated and no communication with financial institutions is performed.
+    Try out your happy paths and not-so happy paths, the sandbox is a great place to get started without transferring actual funds.
```
Comment (tomdalling) [inline source/openapi3/split.yaml]: This is a comma splice, so should be either a semicolon (yuck, old) or a dash (sexy, new).

```suggestion
    Try out your happy paths and not-so happy paths — the sandbox is a great place to get started without transferring actual funds. 
```

File: source/openapi3/split.yaml [hunk 15]
```diff
@@ -737,13 +715,16 @@ info:
...
+    All transactions are simulated and no communication with financial institutions is performed.
 
 
-    The sandbox works on a 1 minute cycle to better illustrate how transactions are received and the lifecyle they go through. In other words, every minute, we simulate communicating with financial institutions and update statuses and events accordingly.
+    The sandbox works on a 1 minute cycle to better illustrate how transactions are received and the lifecyle they go through. 
+    In other words, every minute, we simulate communicating with financial institutions and update statuses and events accordingly.
 
 
-    All 6 digits BSBs are valid in the sandbox with the exception of `100000` which is a place keeper for an invalid BSB. In production, only real BSBs are accepted.
+    All 6 digits BSBs are valid in the sandbox with the exception of `100000` which is a place keeper for an invalid BSB.
```
Comment (tomdalling) [inline source/openapi3/split.yaml]:
```suggestion
    All 6 digits BSBs are valid in the sandbox, with the exception of `100000` which is intended for testing purposes. 
```

File: source/openapi3/split.yaml [hunk 16]
```diff
@@ -895,10 +858,14 @@ info:
...
+    If your integration includes allowing us to pre-fail transactions prior to being processed, you may want to test that your system is handling these events correctly. 
+    A transaction will pre-fail when the available balance of the customers account is less than the amount of the payment being requested. 
+    This is checked during pre-processing just before your debit is sent for processing if there is an active bank connection.
 
 
-    In the Sandbox environment, if the contact you are attempting to debit has a bank connection that was created through our Instant Account Verification feature, the available balance of their bank account will always be `$123.45`. Any payment requests above this amount will pre-fail and any amount less than or equal to this amount will succeed.
+    In the Sandbox environment, if the contact you are attempting to debit has a bank connection that was created through our Instant Account Verification feature, the 
+    available balance of any <strong>Transactional</strong> bank account will always be `$123.45`. Any payment requests above this amount will pre-fail and any amount
```
Comment (tomdalling) [inline source/openapi3/split.yaml]: Double asterisk is bold in Markdown.

```suggestion
    available balance of any **Transactional** bank account will always be `$123.45`. Any payment requests above this amount will pre-fail and any amount 
```

File: source/openapi3/split.yaml [hunk 17]
```diff
@@ -1581,19 +1552,19 @@ tags:
...
-      by using our [Instant Account Verification
-      process](http://help.split.cash/bank-accounts/instant-account-verification-iav)
-      as part of accepting an [Open
-      Agreement](https://help.split.cash/agreements/open-agreement).
+      the result is always the same: A new Agreement. 
+      
+      
+      Additionally, an Open Agreement can be accepted by anybody. This is achieved
+      by using our [Instant Account Verification process]
+      (http://help.split.cash/bank-accounts/instant-account-verification-iav)
```
Comment (tomdalling) [inline source/openapi3/split.yaml:1566]: Does this link look correct in the web page? Some Markdown parsers don't like links being split across two lines like this.

File: source/openapi3/split.yaml (hunk 12 above)
Comment (tomdalling) [reply source/openapi3/split.yaml:4103]: > sometimes

You should have a look in the codebase 😆

**Review by tomdalling — APPROVED** [review]
Looks good to me. A couple of minor suggestions.

**Pull request merged**